src/game/client/draw/w3dmodeldraw.h

```diff
@@ -161,13 +165,19 @@ struct ModelConditionInfo
     }
 
     int Get_Conditions_Count() const { return m_conditionsYesVec.size(); }
+#ifdef GAME_DEBUG_STRUCTS
     Utf8String Get_Definition() const { return m_description; }
+#else
+    Utf8String Get_Definition() const { return "ModelConditionInfo"; }
+#endif
 
+#ifdef GAME_DEBUG_STRUCTS
     Utf8String m_description;
+#endif
     std::vector<BitFlags<MODELCONDITION_COUNT>> m_conditionsYesVec;
     Utf8String m_modelName;
     std::vector<HideShowSubObjInfo> m_hideShowVec;
-    std::vector<Utf8String> m_publicBones;
+    mutable std::vector<Utf8String> m_publicBones;
     Utf8String m_weaponFireFXBoneName[3];
     Utf8String m_weaponRecoilBoneName[3];
     Utf8String m_weaponMuzzleFlashName[3];

```

Comment:
A number of members and loops use this number 3. Can we have an `enum` value for this?

Reply:
fixed.